doc/02-installation.md

```diff
@@ -105,9 +105,20 @@ dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-${OSVER}.n
 <!-- {% if fedora %} -->
 ### Fedora Repository <a id="fedora-repository"></a>
 
+#### Fedora 40 or earlier
+
 ```bash
 dnf install -y 'dnf-command(config-manager)'
 dnf config-manager --add-repo https://packages.icinga.com/fedora/$(. /etc/os-release; echo "$VERSION_ID")/release
+rpm --import https://packages.icinga.com/icinga.key
+```
+
+#### Fedora 41 or later
+
+```bash
+dnf install -y 'dnf-command(config-manager)'
+dnf config-manager addrepo --id=packages.icinga.com --set=name=Icinga --set=baseurl="https://packages.icinga.com/fedora/$(. /etc/os-release; echo "$VERSION_ID")/release"
+rpm --import https://packages.icinga.com/icinga.key
 ```
```

Comment:
> but requires to construct a .repo file

We actually already provide that file on packages.icinga.com. For reference, in our container images, we do the following which works across all Fedora versions:

```sh
rpm --import https://packages.icinga.com/icinga.key
curl -o /etc/yum.repos.d/ICINGA-release.repo https://packages.icinga.com/fedora/ICINGA-release.repo
```

So I'd suggest using that instead as it's a simpler command and needs no version distinction.

Reply:
Good catch, thanks! I have changed the PR accordingly to get rid of the version-dependent cases.